exchanges/gateio/ws_ob_update_manager_test.go

```diff
@@ -37,6 +43,7 @@ func TestProcessOrderbookUpdate(t *testing.T) {
 	})
 	require.NoError(t, err)
 
+	// demonstrate an update is queued
```

Comment:
🐒 Optional nitpick:
🌴 This occurs a few times
* Comment belongs somewhere in a test message below
* Even if it didn't `demonstrate` 👉 `check` or `test` or `ensure`